server/services/transactionsTransformer.go

```diff
@@ -233,6 +237,14 @@ func (transformer *transactionsTransformer) normalTxToRosetta(tx *transaction.Ap
 	}, nil
 }
 
+func (transformer *transactionsTransformer) decideFeePayer(tx *transaction.ApiTransactionResult) string {
+	if len(tx.RelayerAddress) > 0 {
```

Comment:
same here, perhaps extend it to include len of signature as well (maybe even compare they have same length, similar to [this one](https://github.com/multiversx/mx-chain-go/pull/6632/files#diff-a70f47b70b2d542f3eaa01a097d0f6d97e1f49cee567e258d4afc2a79f07af92R6))

Reply:
Changed.